packages/atlas/atlas.config.yml

```diff
@@ -84,6 +84,12 @@ features:
         baseAmount: 250
         actionButtonText: Get referral link
         actionButtonAction: copyReferral
+    widgets: # Widgets on Ypp landing page
+      - title: Discord
```

Comment:
I think this should be "Support".
https://www.figma.com/file/oQqFqdAiPu16eeE2aA5AD5/YouTube-Partner-Program?node-id=1607%3A87168&t=gOOCh3G4LcWCgR2H-4